src/docs/deployment/ios.md

```diff
@@ -193,16 +193,53 @@ In Xcode, configure the app version and build:
 
 Finally, create a build archive and upload it to App Store Connect:
 
-1. Select **Product > Archive** to produce a build archive.
-1. In the sidebar of the Xcode Organizer window, select your iOS app,
-   then select the build archive you just produced.
-1. Click the **Validate App** button. If any issues are reported,
-   address them and produce another build. You can reuse the same
-   build ID until you upload an archive.
-1. After the archive has been successfully validated, click
-   **Distribute App**. You can follow the status of your build in the
-   Activities tab of your app's details page on
-   [App Store Connect][appstoreconnect_login].
+<ol markdown="1">
+<li markdown="1">
+
+Select **Product > Archive** to produce a build archive.
+
+{{site.alert.note}}
+  On Flutter version 1.24.0-6.0 and later you can create an
+  archive by instead running `flutter build ipa`.
```

Comment:
@xster Is this confusing since it produces an `xcarchvie`, not an `ipa`?  `flutter build xcarchive` is an alias of `flutter build ipa` so I can say either.

Reply:
I think ipa is fine since that's the intent. It just happens to have not built one since it doesn't have all the information to take it all the way to the end. 

I forgot the verbiage we put. As long as we print a thing at the end that says "I know you wanted to make an IPA but I can't make one since I don't know how you're gonna distribute it and I can't really ask those choices to you here in the CLI, so why don't you continue the last bit in the GUI by typing open ...", I think it's ok.